doc/extdev/markupapi.rst

```diff
@@ -117,12 +117,30 @@ Both APIs parse the content into a given node. They are used like this::
 
    node = docutils.nodes.paragraph()
    # either
-   from sphinx.ext.autodoc import AutodocReporter
-   self.state.memo.reporter = AutodocReporter(self.result, self.state.memo.reporter)  # override reporter to avoid errors from "include" directive
    nested_parse_with_titles(self.state, self.result, node)
    # or
    self.state.nested_parse(self.result, 0, node)
 
+.. note::
+
+   ``sphinx.util.docutils.switch_source_input()`` allows to change a target file
+   during nested_parse.  It is useful to mixture contents.  For example, ``sphinx.
```

Comment:
``nested_parse``? Also, it is useful to *mix* contents.

Reply:
It seems nested_parse is not a typo. What did you mean?
I will replace second one by "mixed" later.